docs/user/player/advanced/auto-login.md

```diff
@@ -0,0 +1,61 @@
+# Auto Login
+
+:::warning
+
+This feature requires a good understanding of Graasp and might be for the more technically inclined.
+
+:::
+
+In this article we will use the term `itemLogin` to designate the possibility to log a user with a pseudonym.
```

Comment:
```suggestion
In this article we will use the term `itemLogin` to designate the possibility to log a user only with a pseudonym.
```
Or should we have a page about "itemlogin" to reference to? I think it's more "login without an email" than "login with a username"